From eb45086e11566310b33451549a36e5891d84e653 Mon Sep 17 00:00:00 2001
From: Rush Chen <Rush_Chen@wiwynn.com>
Date: Tue, 24 Sep 2024 16:46:40 +0800
Subject: [PATCH 1022/1022] ARM: dts: aspeed: yosemite4: add GPIO I6 to dts

Add BMC GPIO I6 for ALT_SMB_BMC_CPLD2_N.

lore link:
https://lore.kernel.org/lkml/20240826062408.2885174-32-Delphine_CC_Chiu@wiwynn.com/
---
 arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
index 60a3b9aa9748..50745ad54f0d 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-yosemite4.dts
@@ -1998,6 +1998,10 @@ pin_gpio_f7 {
 		gpios = <ASPEED_GPIO(F, 7) GPIO_ACTIVE_LOW>;
 		input;
 	};
+	pin_gpio_i6 {
+		gpios = <ASPEED_GPIO(I, 6) GPIO_ACTIVE_LOW>;
+		input;
+	};
 	pin_gpio_l6 {
 		gpios = <ASPEED_GPIO(L, 6) GPIO_ACTIVE_LOW>;
 		input;
-- 
2.44.2

